airflow_blogpost.txt

```diff
@@ -0,0 +1,110 @@
+Today, we're announcing an integration between Versatile Data Kit and Apache Airflow, a popular workload scheduler. This blog post serves to provide an overview of the relevant parts of these two projects, how they integrate, and how to set up an instance of Airflow which will trigger deployed VDK jobs.
```

Comment:
You need at least one or two pictures (architecture pictures) See https://towardsdatascience.com/from-raw-data-to-a-cleaned-database-a-deep-dive-into-versatile-data-kit-ab5fd992a02e for example on what I mean. 

or https://towardsdatascience.com/an-overview-of-versatile-data-kit-a812cfb26de7